### PR TITLE
Export sequence diagram page with the Vuex store

### PR DESCRIPTION
**Pull request by ahtrotta**
Export the sequence diagram as a page that includes the `store`. This is required to use the sequence diagram as a standalone component.

### CODE REVIEW

File: packages/components/src/pages/SerializedSequence.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <VDiagramSequence :serialized-diagram="serializedDiagram" :interactive="interactive" />
+</template>
+
+<script>
+import VDiagramSequence from '@/components/DiagramSequence.vue';
+import { store } from '@/store/vsCode';
+
+export default {
+  name: 'v-serialized-sequence',
```
Comment (dustinbyrne) [inline packages/components/src/pages/SerializedSequence.vue:10]: I'm not sure about the name, `v-sequence-diagram` seems fine even if it's similar to `v-diagram-sequence`, at least it's a page vs. component.

The fact that the data (?) is serialized seems irrelevant to me.

File: packages/components/src/componentExports.js [hunk 2]
```diff
@@ -9,3 +9,4 @@ export { default as VAnalysisFindings } from '@/pages/AnalysisFindings.vue';
 export { default as VFindingDetails } from '@/pages/FindingDetails.vue';
 export { default as VOpenAppmaps } from '@/pages/install-guide/OpenAppMaps.vue';
 export { default as VSidebarSignIn } from '@/components/SignIn.vue';
+export { default as VSerializedSequence } from '@/pages/SerializedSequence.vue';
```
Comment (dustinbyrne) [inline packages/components/src/componentExports.js:12]: Is the `VDiagramSequence` export used anywhere? Can we drop it?

File: packages/components/src/pages/SerializedSequence.vue (hunk 1 above)
Comment (ahtrotta) [reply packages/components/src/pages/SerializedSequence.vue:10]: Yeah, I had `v-sequence-diagram` originally, but it seemed confusing to have that and `v-diagram-sequence`, but it is a better name so I'll change it back.

File: packages/components/src/componentExports.js (hunk 2 above)
Comment (ahtrotta) [reply packages/components/src/componentExports.js:12]: I'll look into it

File: packages/components/src/pages/SerializedSequence.vue (hunk 1 above)
Comment (kgilpin) [reply packages/components/src/pages/SerializedSequence.vue:10]: Please change the file name too if you do that.

File: packages/cli/src/html/sequenceDiagram.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ async function initializeApp() {
   return new Vue({
     el: '#app',
     render: (h) =>
-      h(VDiagramSequence, {
```
Comment (dustinbyrne) [inline packages/cli/src/html/sequenceDiagram.js:19]: I'd split this into a separate commit due to the fact this is a separate package.

**Review by dustinbyrne — APPROVED** [review]

File: packages/cli/src/html/sequenceDiagram.js (hunk 3 above)
Comment (ahtrotta) [reply packages/cli/src/html/sequenceDiagram.js:19]: :heavy_check_mark:
